vividus-plugin-accessibility/src/main/java/org/vividus/accessibility/engine/HtmlCsAccessibilityTestEngine.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2019-2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.accessibility.engine;
+
+import java.util.List;
+
+import javax.inject.Named;
+
+import org.vividus.accessibility.model.AccessibilityCheckOptions;
+import org.vividus.accessibility.model.AccessibilityViolation;
+import org.vividus.ui.web.action.WebJavascriptActions;
+import org.vividus.util.ResourceUtils;
+import org.vividus.util.json.JsonUtils;
+import org.vividus.util.json.ObjectMapperFactory;
+
+@Named
+public class HtmlCsAccessibilityTestEngine implements AccessibilityTestEngine
+{
+    protected static final String HTML_CS_JS =
+        ResourceUtils.loadResource(HtmlCsAccessibilityTestEngine.class, "HTMLCS.js");
+    protected static final String PA11Y_JS =
+        ResourceUtils.loadResource(HtmlCsAccessibilityTestEngine.class, "pa11y.js");
```

Comment:
what's the reason to keep these constants `protected `?

Reply:
Reuse them in unit tests

Reply:
let's keep them `private`